docs/en_US/NAS/BenchmarksExample.ipynb

```diff
@@ -82,13 +67,28 @@
     "    pprint.pprint(t)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "An architecture of NAS-Bench-101 could be trained more than once. Each element of the returned generator is a dict which contains one of the training results of this architecture including train/valid/test accuracy, training time, number of epochs, etc. The results of NAS-Bench-201 and NDS follow similar formats."
```

Comment:
this architecture => this trial config (architecture + hyper-parameters)

Reply:
fixed.